test/fork/smoke-test/Smoke_Liquidity.t.sol

```diff
@@ -0,0 +1,110 @@
+// SPDX-License-Identifier: BUSL-1.1
+// This code is made available under the terms and conditions of the Business Source License 1.1 (BUSL-1.1).
+// The act of publishing this code is driven by the aim to promote transparency and facilitate its utilization for educational purposes.
+
+pragma solidity 0.8.18;
+
+import { Smoke_Base } from "./Smoke_Base.t.sol";
+import { ForkEnv } from "@hmx-test/fork/bases/ForkEnv.sol";
+
+import "forge-std/console.sol";
+import { ILiquidityHandler } from "@hmx/handlers/interfaces/ILiquidityHandler.sol";
+import { LiquidityHandler } from "@hmx/handlers/LiquidityHandler.sol";
+import { ITradingStaking } from "@hmx/staking/interfaces/ITradingStaking.sol";
+import { IEcoPythCalldataBuilder } from "@hmx/oracles/interfaces/IEcoPythCalldataBuilder.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+contract Smoke_Liquidity is Smoke_Base {
+  ITradingStaking internal hlpStaking = ITradingStaking(0xbE8f8AF5953869222eA8D39F1Be9d03766010B1C);
+
+  function setUp() public virtual override {
+    super.setUp();
+
+  }
+
+  function test_add_liquidity_fork() external {
```

Comment:
Go with the naming convention of
```suggestion
  function testCorrectness_addLiquidity() external {
```
and no need to do internal function if you don't reuse it anywhere.